bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/services/embed/OEmbedJSONResponseImpl.java

```diff
@@ -0,0 +1,143 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2019 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.internal.services.embed;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import com.adobe.cq.wcm.core.components.services.embed.OEmbedResponse;
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(
+    ignoreUnknown = true
+)
+public class OEmbedJSONResponseImpl implements OEmbedResponse {
```

Comment:
all of the standard oEmbed response properties are supported. I think it would make sense to simply link to the response spec [0] in the interface documentation at [1].

[0] - https://oembed.com/#section2.3. 
[1] - https://github.com/adobe/aem-core-wcm-components/pull/718/files#diff-1dc55d963647e188ed576d99e214c556R23.

Reply:
Addressed in 66a217fe789c6a5c